magenta-lib/src/main/scala/magenta/tasks/ASGTasks.scala

```diff
@@ -137,6 +146,8 @@ case class ResumeAlarmNotifications(pkg: DeploymentPackage, stage: Stage, stack:
   lazy val description = "Resuming Alarm Notifications - group will scale on any configured alarms"
 }
 
+class ASGResetException(val throwable: Throwable) extends Throwable("", throwable)
```

Comment:
A better signature for an exception would be (I don't think you need the val and a message is important):
```
class ASGResetException(msg: String, cause: Throwable) extends Throwable(msg, cause)
```